javascripts/controllers/NewCtrl.js

```diff
@@ -1,48 +1,54 @@
 'use strict';
 
 app.controller("NewCtrl", function($location, $rootScope, $scope, ContactService) {
-	$scope.controller = "Hello, NewCtrl. Route:/contacts/new";
-	// console.log("Hello, NewCtrl. Route:/contacts/new");
-
-
-	// const createContact = (contact) => {
-	// 	return {
-	// 		"firstName": contact.firstName,
-	// 		"lastName":  contact.lastName,
-	// 		"phoneHome": contact.phoneHome,
-	// 		"phoneWork": contact.phoneWork,
-	// 		"emailHome": contact.emailHome,
-	// 		"emailWork": contact.emailWork,
-	// 		"notes":	 contact.notes,
-	// 		"uid": $rootScope.uid
-	// 	};
-	// };
+	if($rootScope.flag) {
+		$scope.controller = "Hello, NewCtrl. Route:/contacts/edit/:id";
+		// console.log("Hello, NewCtrl. Route:/contacts/edit/:id");
+	} else {
+		$scope.controller = "Hello, NewCtrl. Route:/contacts/new";
+		// console.log("Hello, NewCtrl. Route:/contacts/new");
+	}
 
 	$scope.saveContact = (contact) => {
 		contact.uid = $rootScope.uid;
-		// console.log(contact);			
+		contact.rating = 0;
```

Comment:
Ken! Why are you rating your friends?? (Rating isn't a key you have in your json or database).

Reply:
Fixed